src/core/server/http/router/route.ts

```diff
@@ -212,7 +212,7 @@ export interface RouteConfig<P, Q, B, Method extends RouteMethod> {
    *   path: 'path/{id}',
    *   validate: {
    *     // handler has access to raw non-validated params in runtime
-   *     params: schema.object({}, { allowUnknowns: true })
+   *     params: schema.object({}, { unknowns: 'allow' })
```

Comment:
Somehow, the corresponding Markdown API doc (at [`docs/development/core/server/kibana-plugin-server.routeconfig.validate.md`, line 52](https://github.com/elastic/kibana/blame/da9cf670f4e9fabe9594213be7c1afe1469fb2bc/docs/development/core/server/kibana-plugin-server.routeconfig.validate.md#L52)) doesn't reflect this change.